addons/mail/static/tests/suggestion/suggestion.test.js

```diff
@@ -330,3 +337,29 @@ test("Mention with @everyone", async () => {
     await click(".o-mail-Composer-suggestion");
     await contains(".o-mail-Composer-input", { value: "@everyone " });
 });
+
+test("@everyone mention adds channel members to the message recipients", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({
+        name: "General",
+        channel_type: "channel",
+        channel_member_ids: [Command.create({ partner_id: serverState.partnerId })],
+    });
+    await start();
+    await openDiscuss(channelId);
+    const messageData = await withUser(serverState.userId, () =>
+        rpc("/mail/message/post", {
+            post_data: {
+                body: "@everyone",
+                message_type: "comment",
+                special_mentions: ["everyone"],
+            },
+            thread_id: channelId,
+            thread_model: "discuss.channel",
+        })
+    );
+    await contains(".o-mail-Message", { text: "@everyone" });
+    expect(
+        messageData["mail.message"][0].recipients.map((r) => r.id).includes(serverState.partnerId)
+    ).toBe(true);
+});
```

Comment:
I'm not sure if there is a better way to write this test. We don't have `partner_ids` in the message_post return value. Partner_ids are added to the message data as `recipients`.

Reply:
Using `recipients` (which is `partner_ids`, we should rename it some day) is the "correct" way to do it if we want to check the "multi-user" part (but this test is not even with multiple users so it's kind of irrelevant).